components/raftstore/src/store/worker/region.rs

```diff
@@ -858,7 +860,12 @@ where
                         for_balance,
                         allow_multi_files_snapshot,
                     );
-                });
+                }).unwrap_or_else(
```

Comment:
Maybe we need notify raftstore that snapshot will not be generated. Otherwise, leader will not retry generate snapshot.

Reply:
I cancel it here, it should be retry generate fast than before.